### DISCUSSION TITLE
after shutting down for a couple of hours.... CHIA has been syncing for DAYS

### DISCUSSION THREAD

**Discussion by freddygusto** [Chia-Network/chia-blockchain#6113] [Support]
I shut down the computer where I'm plotting just for a couple of hours. Now CHIA is taking days to sync back with chain!

Even though it is connected to mainnet, and my internet connection is fine, it is struggling to sync. 
- goes from "syncing 343408/343882" to "not synced"
- the "Time Created" log of the HEADER HASH seems stopped at same time than "Peak time" which is ALWAYS hours behind actual clock

**Comment by freddygusto** [comment]
OK it's synced now

**Comment by BrandtH22** [comment]
@freddygusto , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
